src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -54,17 +55,26 @@ public OrkaAgent(String name, String orkaCredentialsId, String orkaEndpoint, Str
             throws Descriptor.FormException, IOException {
 
         this(name, orkaCredentialsId, orkaEndpoint, vmCredentialsId, vm, node, redirectHost, createNewVMConfig,
-                configName, baseImage, numCPUs, numExecutors, host, port, remoteFS, 
-                useJenkinsProxySettings, ignoreSSLErrors, null);
+                configName, baseImage, numCPUs, numExecutors, host, port, remoteFS,
+                useJenkinsProxySettings, ignoreSSLErrors, null, null);
```

Comment:
`nit`: You can leave this as it was. It will call the second ctor and it will call the 3rd